tests/unit/integrations/kubernetes/orchestrators/test_kubernetes_orchestrator.py

```diff
@@ -43,7 +43,7 @@ def test_kubernetes_orchestrator_remote_stack() -> None:
         password="",
         host="",
         port=0,
-        database="",
+        database="zenml",
```

Comment:
Why was this change neccessary? :-)

Reply:
This PR includes a change unrelated to data validation that checks if the database name respects the format required by SQL. I had to modify the tests too to match.